src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx

```diff
@@ -329,12 +351,21 @@ function LogsComponent({ selectedTab, isDeleted, logSearchTerms, setLogSearchTer
                                                     minHeight: '24px !important',
                                                     cursor: 'pointer',
                                                 }),
+                                                groupHeading: (base) => ({
+                                                    ...base,
+                                                    fontWeight: 600,
+                                                    fontSize: '10px',
+                                                    color: 'var(--n-700)',
+                                                    direction: 'rtl',
+                                                    marginLeft: 0,
```

Comment:
Please check if we have to explicitly provide the text-alignment as well.

Reply:
not required

Reply:
We don't need to